setup.py

```diff
@@ -17,7 +17,7 @@
     'mock>=3.0.5',
     'semver>=2.10.2',
     'IPython==7.16.1',
-    'diskcache==5.1.0',
+    'diskcache==5.4.0',
```

Comment:
Alternatively, we could allow any version higher than 5.1.0, which is similar to most of the other dependencies?
```suggestion
    'diskcache>=5.1.0',
```